### DISCUSSION TITLE
Purchasing without an Account

### DISCUSSION THREAD

**Discussion by LouisLouiseKay** [funmusicplace/mirlo#165] [Ideas]
I think this feature is critical. Purchasing without an account is one of the core features of Bandcamp. It means people aren't necessarily forced to opt into the platform to start supporting artists. Furthermore, one of the other core features, is that we just keep memory of the email used to purchase, so that if someone buys on Mirlo several times using the same email, when they do decide to opt in, they already have a collection of albums waiting for them.

**Comment by simonv3** [comment]
Is it critical for MVP?

**Comment by LouisLouiseKay** [comment]
I'd vote yes on this personally, maybe this is something we can add to the survey to see how other people feel.

The reason I view it as critical is because it helps both onboarding artists (they can share their page and people can support them without them feeling like they're asking to create an account on yet another service) and onboarding people who will generally feal happier interactif with Mirlo if it doesn't require an account to do the basic things (supporting artists). So it makes us look good and this is part of the essential "frictionless payment process" aspect.

**Reply by meljoann** [reply]
Agreed it would be a great inclusive feature, might encourage more support early on. 

Mirlo is gonna really stand out for privacy, if it isn't desperately trying to harvest everyone's emails and data like everyone else!

**Reply by simonv3** [reply]
I think we'll need to get people's e-mail either way to be able to send them a receipt etc (I think Stripe requires it), as well as a link to download their album that exists beyond just the simple purchase flow.

**Comment by tshiunghan** [comment]
Essentially, this feature request is about implementing guest checkout. 

https://www.nngroup.com/articles/login-walls/

We could invite people to create an account after they complete the purchase. 

If Stripe has a Paypal Checkout equivalent, that might be even better.

**Comment by simonv3** [comment]
Should this work for subscriptions too?

**Reply by tshiunghan** [reply]
My sense is yes, to reduce interaction costs. Are there any downsides to treating recurring payments this way? 

Regardless of the transaction, after they've given us their personal info to complete checkout, we could invite them to make a new password to complete account creation.

**Comment by simonv3** [comment]
I'm working on #35 and #112 and the work done for these will actually get this fairly far along.

**Comment by simonv3** [comment]
This is implemented!
